lib/metro-transit.js

```diff
@@ -16,23 +16,21 @@ var MetroTransit = function () {
 MetroTransit.prototype = Object.create(AlexaSkill.prototype);
 MetroTransit.prototype.constructor = MetroTransit;
 
```

Comment:
I'm doing it the same way as in the [sample project](https://github.com/amzn/alexa-skills-kit-js/blob/master/samples/helloWorld/src/index.js#L42).